github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -363,19 +363,18 @@ presubmits:
                   - "true"
               topologyKey: kubernetes.io/hostname
       containers:
-      - image: kubevirtci/bootstrap:v20201119-a5880e0
+      - image: quay.io/kubevirtci/golang:v20210316-d295087
+        env:
+          - name: "TARGET"
+            value: "kind-k8s-sriov-1.17.0"
+          - name: "GIMME_GO_VERSION"
+            value: "1.13.8"
         command:
         - "/usr/local/bin/runner.sh"
         - "/bin/bash"
         - "-c"
         - >
```

Comment:
Since you mentioned that we need add `;` at the end of each command
if we would use `|-` instead, we wont need to add `;`
(Its just a suggestion I understand if you dont want to do it on this PR)

Reply:
Done, used '|' because we we don't care for 'strip'
its not that we have additional empty lines.